jwql/utils/utils.py

```diff
@@ -426,6 +439,69 @@ def check_config(key):
         )
 
 
+def _validate_config(config_file_dict):
+    """Check that the config.json file contains all the needed entries with
+    expected data types
+
+    Parameters
+    -------
```

Comment:
I think `sphinx` will freak out if there are not enough dashes to cover all of the `Parameters` heading

Reply:
Good catch, thank you!